base_api_connection/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module facilitates API connections between Odoo and other web services. It is not usable on its own; rather, it is a low-level module intended to serve as a base for others. An example of such a module is 'project_task_capture'.
```

Comment:
Not sure we should still aim to put this module in the OCA, but `project_task_capture` is certainly not part of it.